api/capi/src/nnstreamer-capi-util.c

```diff
@@ -600,6 +600,38 @@ ml_tensors_data_create_no_alloc (const ml_tensors_info_h info,
   return ML_ERROR_NONE;
 }
 
+/**
+ * @brief Clones the given tensor data frame from the given tensors data. (more info in nnstreamer.h)
+ * @note Memory ptr for data buffer is copied. No new memory for data buffer is allocated.
+ */
+int
+ml_tensors_data_clone_no_alloc (const ml_tensors_data_s * data_src,
+    ml_tensors_data_h * data)
+{
+  ml_tensors_data_s *_data;
+
+  check_feature_state ();
+
+  if (data == NULL || data_src == NULL)
+    return ML_ERROR_INVALID_PARAMETER;
+
+  /* init null */
+  *data = NULL;
+
+  _data = g_new0 (ml_tensors_data_s, 1);
```

Comment:
let's use `g_try_new0` if you wanna handle out-of-memory.

Reply:
yes, thats a bug, updated.